action.yml

```
@@ -0,0 +1,71 @@
+name: "Create Escrow"
+description: "Launch a campaign escrow with provided parameters"
+author: "Hu-Fi"
+inputs:
+  CHAIN_ID:                   { required: true }
+  DAILY_VOLUME_TARGET:        { required: true }
+  DURATION:                   { required: true }
+  EXCHANGE_NAME:              { required: true }
+  EXCHANGE_ORACLE_ADDRESS:    { required: true }
+  EXCHANGE_ORACLE_FEE:        { required: true }
+  GAS_WARN_THRESHOLD:         { required: true }
+  RECORDING_ORACLE_ADDRESS:   { required: true }
+  RECORDING_ORACLE_FEE:       { required: true }
+  REPUTATION_ORACLE_ADDRESS:  { required: true }
+  REPUTATION_ORACLE_FEE:      { required: true }
+  REWARD_AMOUNT:              { required: true }
+  REWARD_TOKEN:               { required: true }
+  START_DELAY:                { required: true }
+  SYMBOL:                     { required: true }
+  NODE_VERSION_FILE:
+    required: false
+    default: ".nvmrc"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Read Node version from action’s .nvmrc
+      id: nvm
+      shell: bash
+      run: |
+        echo "GITHUB_ACTION_PATH=$GITHUB_ACTION_PATH"
+        cat "$GITHUB_ACTION_PATH/.nvmrc"
+        ver="$(tr -d ' \r\n' < "$GITHUB_ACTION_PATH/.nvmrc")"
+        echo "version=$ver" >> "$GITHUB_OUTPUT"
+    - name: Set up Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ steps.nvm.outputs.version }}
```

If define action path above this step - you probably can use `.nvmrc` path as below
```suggestion
      with:
          node-version-file: .nvmrc
          cache: yarn
          cache-dependency-path: '**/yarn.lock'
```